src/arviz_plots/plots/parallel_plot.py

```diff
@@ -0,0 +1,238 @@
+    plot_bknd = import_module(f".backend.{backend}", package="arviz_plots")
+    if plot_collection is None:
+        pc_kwargs.setdefault("cols", [dim for dim in data.dims if dim not in new_sample_dims])
+        pc_kwargs["figure_kwargs"] = pc_kwargs.get("figure_kwargs", {}).copy()
+        pc_kwargs = set_wrap_layout(pc_kwargs, plot_bknd, data)
+        pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+        plot_collection = PlotCollection.wrap(
+            data.to_dataset(),
+            backend=backend,
+            **pc_kwargs,
+        )
+
+    if aes_by_visuals is None:
+        aes_by_visuals = {}
+    else:
+        aes_by_visuals = aes_by_visuals.copy()
+    aes_by_visuals.setdefault("line", plot_collection.aes_set)
+    aes_by_visuals.setdefault("divergence", plot_collection.aes_set)
+
+    line_kwargs = copy(visuals.get("line", {}))
+    if line_kwargs is not False:
+        _, line_aes, line_ignore = filter_aes(
+            plot_collection, aes_by_visuals, "line", new_sample_dims
+        )
+        if "color" not in line_aes:
+            colors = plot_bknd.get_default_aes("color", 1, {})
+            line_kwargs.setdefault("color", colors[0])
+        if "alpha" not in line_aes:
+            line_kwargs.setdefault("alpha", 0.05)
+        plot_collection.map(
+            multiple_lines,
+            "line",
+            data=data,
+            xname="label",
+            ignore_aes=line_ignore,
+            store_artist=backend == "none",
+            **line_kwargs,
+        )
+
+    # divergence
+    div_kwargs = copy(visuals.get("divergence", False))
+    if div_kwargs is True:
+        div_kwargs = {}
+    sample_stats = get_group(dt, "sample_stats", allow_missing=True)
+    if (
+        div_kwargs is not False
+        and sample_stats is not None
+        and "diverging" in sample_stats.data_vars
+        and np.any(sample_stats.diverging)
+    ):
+        divergence_mask = dt.sample_stats.diverging
+        stacked_mask = divergence_mask.stack(sample=sample_dims)
+        aligned_mask = stacked_mask.sel(sample=data.coords["sample"])
+        divergent_data = data.where(aligned_mask)
+        _, div_aes, div_ignore = filter_aes(
+            plot_collection, aes_by_visuals, "divergence", new_sample_dims
+        )
+        if "color" not in div_aes:
+            div_kwargs.setdefault("color", "black")
+        if "alpha" not in div_aes:
+            div_kwargs.setdefault("alpha", 0.1)
+        plot_collection.map(
+            multiple_lines,
+            "divergence",
+            data=divergent_data,
+            xname="label",
+            ignore_aes=div_ignore,
+            store_artist=backend == "none",
+            **div_kwargs,
+        )
+
+    # x-axis label
+    x_labels = data.coords["label"].values
+    # process labels to get vertical alignment
+    processed_labels = [re.sub(r"(.*?)\[(.*?)\]", r"\1\n\2", label) for label in x_labels]
+    total = len(data.values[0])
```

Comment:
instead of `todal` devine `x_values = np.arange(len(x_labels))`, move to before any plotting and send `x_values` to the multiple_lines function as `x`

try numpy otherwise `xr.DataArray(x_values, dims=["label"])`